v5/tutorials/walkthrough/uploading.rst

```diff
@@ -1,5 +0,0 @@
-=========

```

Comment:
are we waiting for interactive UI to bring this back?

Reply:
I dunno. I don't know what to put in here. You click da upload button in the menu or on the toolbar.

Reply:
I don't disagree, but there have been a surprising (to me) number of people looking for "the uploading docs."

at one point we had discussed adding some guide with screenshots of the editor (possibly that just meant updating the clawbot guide)